client/src/pages/Login.vue

```diff
@@ -0,0 +1,236 @@
+<template>
+    <v-container no-gutters fluid fill-height class="pa-0 ma-0">
+      <v-row class="d-flex flex-column align-center justify-center">
+        <v-col class="pa-0 ma-0">
+          <v-sheet
+            class="section d-flex align-center"
+            style="padding-bottom: 200px"
+          >
+            <!-- Right side for login component -->
+            <v-col
+              cols="12"
+              xs="12"
+              sm="10"
+              md="10"
+              lg="4"
+              xl="3"
+              xxl="3"
+              class="d-flex flex-column justify-start align-start"
+            >
+                <h1 class="login-text mx-auto">Log In</h1>
+                <div class="text-input-wrapper">
+                    <input class="duolingo-text-input" v-model="email" data-test="email-input" type="text" placeholder="Email or username">
+                    <button @click="clearEmailOrUser" class="cancel-email-input-button _1eJKW _16r-S NF_yB tV2C-" tabindex="-1" type="button">
+                        <img src="https://d35aaqx5ub95lt.cloudfront.net/vendor/8857e57af4c38e2caab2df3d086202da.svg">
```

Comment:
Can we download this and keep it in assets? If it ever changes I think it would be best to have our own copy

Reply:
Put it in the assets folder